Examples/Tests/PEC/inputs_field_PEC_3d

```diff
@@ -0,0 +1,56 @@
+# Set-up to test the PEC Boundary condition for the fields
+# Constructive interference between the incident and reflected wave result in a
+# standing wave.
+
+# max step
+max_step = 125
+
+# numver of grid points
+amr.n_cell = 32 32 256
+
+# Maximum allowable size of each subdomain
+amr.max_grid_size = 1024
+amr.blocking_factor = 32
+
+amr.max_level = 0
+
+# Geometry
+geometry.coord_sys   = 0                     # 0: Cartesian
+geometry.prob_lo     =  -8.e-6 -8.e-6 -4.e-6
+geometry.prob_hi     =   8.e-6  8.e-6  4.e-6
+
+# Boundary condition
+boundary.field_lo = periodic periodic pec
+boundary.field_hi = periodic periodic pec
+
+warpx.serialize_ics = 1
+
+# Verbosity
+warpx.verbose = 1
+
+# Algorithms
+algo.current_deposition = esirkepov
+# CFL
+warpx.cfl = 0.9
+
+
+my_constants.z1 = -2.e-6
+my_constants.z2 = 2.e-6
+my_constants.pi = 3.14159265359
```

Comment:
Note that pi and clight are already defined and are not needed here.

Reply:
@dpgrote Thank you. Does this mean, I dont need to define them as constants. And when we have `pi` and `clight` in the parser expression, the code will use the defined values. Also, should it be `c` or `clight` in the expression then?

Reply:
Done!